examples/account/unchecked.rs

```diff
@@ -0,0 +1,70 @@
+// Copyright 2020-2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+//! cargo run --example account_advanced
+
+use std::path::PathBuf;
+
+use identity::account::Account;
+use identity::account::AccountStorage;
+use identity::account::IdentitySetup;
+use identity::account::Result;
+use identity::core::Timestamp;
+use identity::iota::ExplorerUrl;
+use identity::iota::IotaDID;
+use identity::prelude::IotaDocument;
+
+#[tokio::main]
+async fn main() -> Result<()> {
+  pretty_env_logger::init();
+
+  // Sets the location and password for the Stronghold
+  //
+  // Stronghold is an encrypted file that manages private keys.
+  // It implements best practices for security and is the recommended way of handling private keys.
+  let stronghold_path: PathBuf = "./example-strong.hodl".into();
+  let password: String = "my-password".into();
+
+  // Create a new identity using Stronghold as local storage.
+  //
+  // The creation step generates a keypair, builds an identity
+  // and publishes it to the IOTA mainnet.
+  let mut account: Account = Account::builder()
+    .storage(AccountStorage::Stronghold(stronghold_path, Some(password), None))
+    .create_identity(IdentitySetup::default())
+    .await?;
+
+  // Get a copy of the document this account manages.
+  // We will apply updates to the document, and overwrite the account's current document.
+  let mut document: IotaDocument = account.document().clone();
+
+  // Create a timestamp that points to 24 hours (= 86400 seconds) in the future.
+  let timestamp: Timestamp = Timestamp::from_unix(Timestamp::now_utc().to_unix() + 86400);
+
+  // Set a custom timestamp.
+  document.set_updated(timestamp);
+
+  // Set a custom property.
+  document.properties_mut().insert("key".into(), "value".into());
+
+  // Update the identity without validation by the account, and publish the result (depending on autopublish).
+  // The responsibility is on us to provide a valid document, which the account can continue to use.
+  // Failing to do so can corrupt the identity; use with caution!
+  account.update_document_unchecked(document).await?;
```

Comment:
Optional: minor changes to the example comments.

```suggestion
  // Get a copy of the document this account manages.
  // We will apply updates to the document and overwrite the account's current document.
  let mut document: IotaDocument = account.document().clone();

  // Add a custom property to the document.
  document.properties_mut().insert("myCustomPropertyKey".into(), "value".into());

  // Override the updated field timestamp to 24 hours (= 86400 seconds) in the future,
  // because we can. This is usually set automatically by Account::update_identity.
  let timestamp: Timestamp = Timestamp::from_unix(Timestamp::now_utc().to_unix() + 86400);
  document.set_updated(timestamp);

  // Update the identity without validation and publish the result to the Tangle 
  // (depending on the account's autopublish setting).
  // The responsibility is on the caller to provide a valid document which the account 
  // can continue to use. Failing to do so can corrupt the identity; use with caution!
  account.update_document_unchecked(document).await?;
```